lib/standalone.mjs

```diff
@@ -83,6 +83,16 @@ export default async function standalone({
 	let formatterFunction;
 
 	try {
+		/** @see https://github.com/stylelint/stylelint/issues/7447 */
+		if (isString(formatter) && formatter === 'github') {
+			process.emitWarning(
```

Comment:
medium

Use `emitDeprecationWarning` util.

Reply:
Added a new code to `emitDeprecationWarning`:  `GITHUB_FORMATTER: '004'`